evap/staff/templates/staff_evaluation_textanswers_quick.html

```diff
@@ -21,6 +21,7 @@ <h5 class="modal-title" id="hotkeys-modal-title">Hotkeys</h5>
                     <tr><td><kbd>&#x21b2;</kbd></td><td>{% trans 'Review next evaluation' %}</td></tr>
                     <tr><td><kbd>N</kbd></td><td>{% trans 'Show all again' %}</td></tr>
                     <tr><td><kbd>M</kbd></td><td>{% trans 'Show unreviewed' %}</td></tr>
+                    <tr><td><kbd>S</kbd></td><td>{% trans 'Skip evaluation' %}</td></tr>
```

Comment:
please move the new entry between "Review next evaluation" and "Show all again", then the order of the hotkeys and buttons is consistent.